cmds/core/uptime/uptime.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Get the time the machine has been up
+// Synopsis:
+//     uptime
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+)
+
+// loadavg takes in the contents of proc/loadavg,it then extracts and returns the three load averages as a string
+func loadavg(contents string) (loadaverage string, err error) {
+	loadavg := strings.Fields(contents)
+	if len(loadavg) < 3 {
+		return "", errors.New("invalid contents:the contents of proc/loadavg we are trying to process contain less than the required 3 loadavgs ")
```

Comment:
Missed this earlier, but it's always useful to print what was actually read.

Reply:
Done